sql/ban-collectives.sql

```diff
@@ -110,6 +110,12 @@ WITH banned_collectives AS (
   USING       deleted_users
   WHERE       n."UserId" = deleted_users.id
   RETURNING   n.id
+), deleted_recurring_expenses AS (
+  -- Delete Recurring Expenses 
+  DELETE FROM "RecurringExpenses" re
+  USING       deleted_profiles
```

Comment:
Soft delete should be preferred in this script whenever possible

```suggestion
  UPDATE ONLY "RecurringExpenses" re SET "deletedAt" = NOW()
  FROM       deleted_profiles
```